.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "3.8"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+# command to run tests
+script:
+  # First sample run test on PIP version
+  - pytest tests/test_src.py
+notifications:
+  email:
+    - skizza8@gmail.com
```

Comment:
can we remove the notifications bit, you wouldn't want to receive a notification about each and every build

Reply:
Done.